### DISCUSSION TITLE
Is there a configuration to enable/disable auto-dating file names?

### DISCUSSION THREAD

**Discussion by rshmhrj** [estruyf/vscode-front-matter#452] [Q&A] [answered]
Hi all!
First -- thanks for all the work on this awesome extension.  I have found it extremely useful in setting up and maintaining the site.  The more I use it, and the more familiar I am getting with the settings, the better it gets!

### Context:
I have two sections in my Hugo site: `posts` and `references`.  From the front matter dashboard, clicking `create content` pops up a question asking which folder to add the content to, what to title it, and then it auto creates the file based on a template. 

 **It also slugifies the title and prepends the date for the filename.**

For `posts`, I like the auto dating behavior.  For `references`, I want to disable it.

### Question
**Is there a config setting to selectively enable/disable the auto-dating based on which folder is selected?** 

Obviously, the "workaround" is to create it and then manually delete the date portion of the filename (which takes 2 seconds), but just wondering if such a feature / configuration exists.  I searched through the existing issues and [discussions](https://github.com/estruyf/vscode-front-matter/discussions/252) , but didn't find anything specifically related to the folder choice.

Thanks again!

**Comment by estruyf** [comment, marked as answer]
@rshmhrj thank you for starting this discussion. Currently, the filename prefix is configured globally, which means it gets added to all the pages. 

We could work on an enhancement where you can configure `prefix` values per content page folder. That way, you can override the global configuration and have it defined on the content folder level.

**Reply by rshmhrj** [reply]
Thanks @estruyf!  Tested out https://github.com/estruyf/vscode-front-matter/issues/458 in the Beta build and it works perfectly!
